src/toTransform.js

```diff
@@ -169,17 +169,23 @@
     var position = 'auto';
     if ('offsetPosition' in properties) {
       position = internalScope.offsetPositionAnchorParse(properties['offsetPosition']);
+      if (!position) {
+        position = 'auto';
+      }
     }
 
     var anchor = 'auto';
     if ('offsetAnchor' in properties) {
       anchor = internalScope.offsetPositionAnchorParse(properties['offsetAnchor']);
+      if (!anchor) {
+        anchor = 'auto';
+      }
     }
 
     var transformOrigin = window.getComputedStyle(element).transformOrigin;
     transformOrigin = transformOrigin.split(/\s+/).map(internalScope.offsetDistanceParse);
 
-    if (!anchor || anchor === 'auto') {
+    if (anchor === 'auto') {
       if (!properties['offsetPath'] || properties['offsetPath'] === 'none') {
```

Comment:
it's a bit confusing that an offset-path of none is checked here, but not an offset-position of auto (that seems to be handled on line 214 now). I would suggest adding to this if condition  && position != 'auto'. That guarantees that after this block anchor will no longer be auto and will have an x and y value (whether through position of transformOrigin)